repos/public-user/src/Settings/PublicUserSettings.php

```diff
@@ -15,7 +14,22 @@ class PublicUserSettings
     const ACTIVATION_AUTOMATIC = 'public-user-automatic-activation';
     const LOGIN_REDIRECT = 'public-user-login-redirect';
     const DEFAULT_NEW_ROLE = 'public-user-registration-role';
-
+    const INVITE_ONLY = 'public-user-invite-only';
+
+    const ADDITIONAL_ROLES = [
+        // Original Omeka roles
+        'viewer' => 'Viewer',
+        'editor' => 'Editor',
+        'admin' => 'Admin',
+        // Can transcribe tasks assigned to them.
+        'limited-transcriber' => 'Limited Transcriber',
+        // Can transcribe and review tasks assigned to them.
+        'limited-reviewer' => 'Limited Reviewer',
+        // Can transcribe anything
+        'transcriber' => 'Transcriber',
+        // Can transcribe and review anything.
+        'reviewer' => 'Reviewer',
+    ];
```

Comment:
I thought about making this non-constant, but I think we want this to be a constant. We're not looking for infinitely expandable user roles at the moment, we want stable predictable roles that we can build customisable functionality around.